index.js

```diff
@@ -18,7 +20,17 @@ github.authenticate({
     token: process.env.FORKED_TOKEN,
 })
 
-const packageJson = shell.cat('package.json')
+const args = process.argv.slice(2)
+if (args.indexOf('-h') !== -1 || args.indexOf('--help') !== -1) {
+  console.log(`  Easy-peasy usage:
+    1. Navigate to directory of dependency you want to fork.
+    2. run \`fork\`
+
+  Lemon-squeezy usage:
+    1. From your project’s root directory, run \`fork dependencyName\`.`)
```

Comment:
Is this comment on the right line?

In any case, I _think_ you mean using a glob to search for whatever your first argument is, recursively down any number of directories that might exist within the cwd, rather than the rigid set of directories I’m searching on now. Is that correct?